temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt

```diff
@@ -501,7 +501,7 @@ fun KeypadCircleButton(
 ) {
     Text(
         modifier = circleButtonModifier(size = btnSize, onClick = onClick)
-            .padding(top = 10.dp)
+            .padding(13.dp)
```

Comment:
Use `top = 13.dp`. Otherwise the app may break.

Reply:
Required changes Made Successfully